models/issue_tracked_time.go

```diff
@@ -299,10 +303,8 @@ func deleteTime(e Engine, t *TrackedTime) error {
 
 // GetTrackedTimeByID returns raw TrackedTime without loading attributes by id
 func GetTrackedTimeByID(id int64) (*TrackedTime, error) {
-	time := &TrackedTime{
-		ID: id,
-	}
-	has, err := x.Get(time)
+	time := new(TrackedTime)
+	has, err := x.ID(id).Get(time)
```

Comment:
This is OK, but I wonder if there's some specific reason for the change, or it's just for tidiness.

Reply:
This will always generate `WHERE id =?`, but if `id == 0`, FindByExample will not generate any query conditions. So I will give this a ❤️